main.js

```diff
@@ -0,0 +1,30 @@
+// Get array data from api to variable
+function fetchData(cb, func) {
+  $.ajax({
+    url: "https://randomuser.me/api/?results=15",
+    success: function (response) {
+      usersArr = response
```

Comment:
You can drop the `Arr` at the end of the variable name. If it's in the plural it is understood that it is an array (most of the time).